transaction-service/transaction-module/src/main/kotlin/com/ajaxsystems/application/service/CurrencyExchangeService.kt

```diff
@@ -1,7 +1,7 @@
-package ua.com.foxminded.restClient.service
+package com.ajaxsystems.application.service
 
+import com.ajaxsystems.domain.dto.TransactionDto
 import java.util.Currency
-import ua.com.foxminded.restClient.dto.TransactionDto
 
 interface CurrencyExchangeService {
```

Comment:
let's call this class CurrencyExchangeInPort and remove Impl suffix from service implementation

Reply:
I will name it out port because this service use rate-module